pkg/network/ovn_kubernetes.go

```diff
@@ -910,7 +1126,18 @@ func fillOVNKubernetesDefaults(conf, previous *operv1.NetworkSpec, hostMTU int)
 		var syslogfacility string = "local0"
 		sc.PolicyAuditConfig.SyslogFacility = syslogfacility
 	}
-
+	if previous != nil && previous.DefaultNetwork.OVNKubernetesConfig != nil &&
```

Comment:
@mohammadheib per our conversation I think we should move this to the render phase, rather than fill-defaults. That way we can report Progressing status too.